src/stop.sh

```diff
@@ -161,13 +161,23 @@ if [ $stop_all -eq 1 ]; then
         do_killcephadm
     fi
 
+    # killing processes
     for p in $ceph_osd ceph-mon ceph-mds ceph-mgr radosgw lt-radosgw apache2 ganesha.nfsd ; do
-        for try in 0 1 1 1 1 ; do
```

Comment:
if osd takes longer than 1 seconds to quit, can we use the same fibonacci sequence in the inner loop. but send different signals, like
```sh
for p in $ceph_osd ceph-mon ...; do
  for try in 1 1 2 3 5 8; do
    case $try in
      0) send_signal=SIGTERM;
      8) echo "WARNING: $p failed to stop before timeout. terminating it!"; send_signal=SIGTERM;;
      *) send_signal=0;;
    esac
    if ! pkill -$send_signal -u $MYUID $p; then
      break
    fi
    sleep $try
  done
done
```

    elif [[